assume/scenario/loader_csv.py

```diff
@@ -54,6 +58,8 @@ def load_file(
     df = None
 
     if file_name in config:
```

Comment:
having a file_name = None would differ from having it unset.
Is this wanted?

I would suggest to just check: `if config.get(file_name)` instead?

Reply:
The idea is that I can have one set of inputs, but disable, for example, loading storage units, so in my config, I would say storage_units: null

Reply:
sounds good